src/components/MobileWorkerPool.tsx

```
@@ -0,0 +1,234 @@
+          <span className="mobile-worker-pool-fab-icon">👷</span>
+          <span className="mobile-worker-pool-fab-badge">{workers.length}</span>
+        </button>
+      )}
+
+      {isOpen && (
+        <>
+          <div
+            className="mobile-worker-pool-overlay"
+            data-testid="bottom-sheet-overlay"
+            role="button"
+            tabIndex={0}
```

Biome (2.3.13)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)